src/app/components/about/about.js

```diff
@@ -0,0 +1,70 @@
+import Link from "next/link"
+
+export default function AboutSection() {
+    return (
+        <div >
+            <div >
+                <h1></h1>
+                <h3 className={`text-center mt-8 ml-10 mr-10 text-sky-300 text-4xl md:text-6xl  mb-2 mt-2`}>About Us</h3>
+                <p className={`ml-10 mr-10 font-sans flex flex-row mt-8 gap-2`} >
+                    The Hour of Code started as a one-hour introduction to computer science, designed to demystify "code", to show that anybody can learn the basics, and to broaden participation in the field of computer science. It has since become a worldwide effort to celebrate computer science, starting with 1-hour coding activities but expanding to all sorts of community efforts. Howard County's version of Hour of Code is made by students, for students.
+                    <br></br><br></br><br></br><br></br>
+                </p>
+                <div class="flex flex-row space-x-4 ml-10 mr-10">
+                    <div class="flex-grow-2 font-mono">
+                        <h4 className={`mt-8 mr-20 text-indigo-400 text-3xl md:text-5xl  mb-2 mt-2`}>Leaderboard</h4>
+                        <p class="font-sans mt-8 gap">We aim to provide aspiring computer scientists with an opportunity for skill development and friendly competition. Visit the leaderboard to check out how your high school is doing! All Howard County schools and corresponding points will be displayed here.</p>
+                        <br></br>
+                        <Link class=" btn-secondary" href={"/leaderboard"}> View Leaderboard </Link>
+                    </div>
+                    
+                        <div className={`flex-grow w-full h-full text-slate-300 bg-gray-900 rounded-md border-gray-800 border-2 overflow-hidden mr-10`}>
+                            <table className={`w-full h-full table-auto`}>
```

Comment:
it would be better to just use the leaderboard component and pass in dummy data instead of duplicating a ton of code here

Reply:
done